libpod/inspect_data.go

```diff
@@ -0,0 +1,103 @@
+package libpod
+
+import (
+	"time"
+
+	digest "github.com/opencontainers/go-digest"
+	"github.com/opencontainers/image-spec/specs-go/v1"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/projectatomic/libpod/libpod/driver"
+)
+
+// ContainerInspectData handles the data used when inspecting a container
+type ContainerInspectData struct {
+	ID              string           `json:"ID"`
+	Created         time.Time        `json:"Created"`
+	Path            string           `json:"Path"`
+	Args            []string         `json:"Args"`
+	State           *CtrState        `json:"State"`
+	ImageID         string           `json:"Image"`
+	ImageName       string           `json:"ImageName"`
+	ResolvConfPath  string           `json:"ResolvConfPath"`
+	HostnamePath    string           `json:"HostnamePath"` //TODO
+	HostsPath       string           `json:"HostsPath"`    //TODO
+	StaticDir       string           `json:"StaticDir"`
+	LogPath         string           `json:"LogPath"`
+	Name            string           `json:"Name"`
+	RestartCount    int32            `json:"RestartCount"` //TODO
+	Driver          string           `json:"Driver"`
+	MountLabel      string           `json:"MountLabel"`
+	ProcessLabel    string           `json:"ProcessLabel"`
+	AppArmorProfile string           `json:"AppArmorProfile"`
+	ExecIDs         []string         `json:"ExecIDs"` //TODO
+	GraphDriver     *driver.Data     `json:"GraphDriver"`
+	SizeRw          int64            `json:"SizeRw,omitempty"`
+	SizeRootFs      int64            `json:"SizeRootFs,omitempty"`
+	Mounts          []specs.Mount    `json:"Mounts"`
+	NetworkSettings *NetworkSettings `json:"NetworkSettings"` //TODO
+}
+
+// CtrState represents the state of a container.
+type CtrState struct {
```

Comment:
Nit: can you rename this to ContainerState to match ContainerInspectData?

Reply:
There is already a ContainerState in container.go, should I rename it to ContainerInspectState?

Reply:
Yeah, that sounds fine

Reply:
fixed